templates/node.js-express-webapp-on-linux.yml

```diff
@@ -6,35 +6,74 @@
 trigger:
 - master
 
-pool:
-  vmImage: 'ubuntu-latest'
-

```

Comment:
Templates should be multi-stage. CI jobs in build stage and CD jobs in deploy stage

Reply:
Changing the same in this PR. Am I missing something ?